crates/bevy_reflect/Cargo.toml

```diff
@@ -14,6 +14,7 @@ bevy = ["glam", "smallvec"]
 
 [dependencies]
 # bevy
+bevy_math = { path = "../bevy_math", version = "0.9.0-dev" }
```

Comment:
bevy_reflect is trying to be as Bevy agnostic as possible. Glam is behind a feature, could you also put bevy_math as optional and enabled with the `bevy` feature?

Reply:
Done.